### PR TITLE
Upgrade Gravatar module to v9, make installable, and add tests

### PR DESCRIPTION
**Pull request by lasley**
Hi All!

This PR removes deprecated `api.one` in the Gravatar module, adds tests, and a few styling changes in the ReadMe.

Not sure how to get Transifex to pass. I tried signing up, but it's still complaining so I ignored it.

### CODE REVIEW

**Comment by danimaribeiro** [comment]
:+1: Tested on runbot.

File: base_user_gravatar/tests/test_res_users.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
```
Comment (pedrobaeza) [inline base_user_gravatar/tests/test_res_users.py]: Please use short headers like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

File: base_user_gravatar/tests/test_res_users.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        self.url = 'http://www.gravatar.com/avatar/{}?s=200'
+
+    def _test_record(self, ):
+        partner_id = self.env['res.partner'].create(self.partner_vals)
+        self.vals['partner_id'] = partner_id.id
+        return self.env['res.users'].create(self.vals)
+
+    @mock.patch('%s.urllib2' % MODULE_LOCATION)
+    def test_get_gravatar_base64_opens_correct_uri(self, mk, ):
+        ''' Test that gravatar is pinged for image '''
```
Comment (pedrobaeza) [inline base_user_gravatar/tests/test_res_users.py]: Use double quote (`"""`) for docstring

**Comment by lasley** [comment]
Thanks for the review @danimaribeiro & @pedrobaeza - changes made per suggestions

**Comment by pedrobaeza** [comment]
:+1:

**Comment by sebalix** [comment]
:+1:

**Pull request merged**